src/px-script/main/rangeslider/index.js

```diff
@@ -12,60 +18,57 @@ const _writeStyle = obj => {
     inlineStyle.textContent = styleText;
 };
 
-const rangeslider = (() => {
-    const init = () => {
-        const rangeContainers = document.querySelectorAll(".rangeslider");
-        const userBrowser = navigator.userAgent;
-        const inlineStyle = document.createElement("style");
-        const inlineStyleContent = [];
-        const isBrowserChrome = userBrowser.indexOf("Chrome") > -1;
-
-        if (rangeContainers.length > 0) {
-            if (isBrowserChrome) {
-                document.body.appendChild(inlineStyle);
-            }
+const init = ids => {
+    const rangeContainers = ids || ids === "" ? getElementsByIds(ids, "rangeslider") : document.querySelectorAll(SELECTORS.RANGESLIDER);
+    const inlineStyle = document.createElement("style");
+    const inlineStyleContent = [];
+    const isBrowserChrome = navigator.userAgent.indexOf("Chrome") > -1;
 
-            rangeContainers.forEach((rangeContainer, i) => {
-                const input = rangeContainer.querySelector("input[type=range]");
-                const valueSpan = rangeContainer.querySelector("span[data-rs-value]");
+    if (rangeContainers.length > 0) {
+        if (isBrowserChrome) {
```

Comment:
Why do we handle chrome differently?

Reply:
Because chrome has some weird behavior when it comes to filling just the part of the rangeslider that is "behind" the rangeslider thumb. @erikhallaraaker may have more info. Tried to google it but couldn't find anything specifically related to this issue.